assets/enums/material.json

```
@@ -39,6 +39,7 @@
         "latrine",
         "skull",
         "shell",
-        "skin of leg"
+        "skin of leg",
+        "hair"
```

```suggestion
        "strand of hair"
```

https://www.ebi.ac.uk/ols/ontologies/uberon/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0001037